bin/simplifiedAdminPolygons.pl

```diff
@@ -61,6 +61,19 @@
 out;
 ---EOF---
 
+}elsif( $opt{'ds'} eq 'AN134cd' ){
+    $aTerr = getTerritories();
+    $COMPUTATION_ZOOM = 6;
+    $OUTFILE_NAME = 'ogf_polygons';
+    $ADMIN_RELATION_QUERY = << '---EOF---';
+[timeout:1800][maxsize:4294967296];
+(
+  (relation["boundary"="administrative"]["ogf:id"~"^AN134[cd]$"];);
+  >;
+);
+out;
+---EOF---
+

```

Comment:
AN134c and AN134d are randomly (more often than not) causing "polygon not closed" errors, I've added this data source argument to allow focused testing to troubleshoot this later